src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
```

Comment:
`h-screen` makes this section way too long and the top padding is too much, especially because it doesn't fully fit on the page due to the height of the navbar. Just set a vertical padding; the viewport doesn't need to only contain this section, so it's okay if the content below the image is visible before you scroll down.